tbmodels/_tb_model.py

```diff
@@ -717,6 +772,90 @@ def remap_hoppings(hop_entries):
 
             return cls.from_hop_list(size=num_wann, hop_list=hop_entries, **kwargs)
 
+    @classmethod  # noqa: MC0001
+    def from_wannier_tb_files(  # pylint: disable=too-many-locals
+        cls,
+        *,
+        tb_file: str,
+        wsvec_file: str,
+        **kwargs,
+    ) -> Model:
+        """
+        Create a :class:`.Model` instance from Wannier90 output files.
+
+        Parameters
+        ----------
+        tb_file :
+            Path of the ``*_tb.dat`` file. Together with the
+            ``*_wsvec.dat`` file, this determines the hopping terms.
+        wsvec_file :
+            Path of the ``*_wsvec.dat`` file. This file determines the
+            remapping of hopping terms when ``use_ws_distance`` is used
+            in the Wannier90 calculation.
+        kwargs :
+            :class:`.Model` keyword arguments.
+        """
+
+        if "uc" in kwargs:
+            raise ValueError(
+                "Ambiguous unit cell: It can be given either via 'uc' or the 'tb_file' keywords, but not both."
+            )
+        if "pos" in kwargs:
+            raise ValueError(
+                "Ambiguous orbital positions: The positions can be given either via the 'pos' or the 'xyz_file' keywords, but not both."
```

Comment:
```suggestion
                "Ambiguous orbital positions: The positions can be given either via the 'pos' or the 'tb_file' keywords, but not both."
```
Is this a typo, or is the `xyz_file` also consumed here?